spec/support/evm_spec_helper.rb

```diff
@@ -12,6 +12,15 @@
 module EvmSpecHelper
   extend RSpec::Mocks::ExampleMethods
 
+  module EmsFolder
```

Comment:
This name sort of overloads the model name and can be confusing.  Let's change it to maybe something like `EmsMetadataHelper` implying it helps with setting up the ems_metadata trees.